src/pages/gown.js

```diff
@@ -0,0 +1,375 @@
+  border-bottom: 48px solid ${props => props.theme.colors.primaryPink};
+  @media (min-width: ${props => props.theme.breakpoints.tablet}) {
+  display: none;
+}
+`;
+
+const DifferentAnglesContainer = styled.div`
+display: none;
+@media (min-width: ${props => props.theme.breakpoints.tablet}) {
+  display: flex;
+  margin-top: 30px;
+  margin-left: 10%;
+  border-bottom: 200px solid ${props => props.theme.colors.backgroundGray};
+}
+`
+
+const DifferentAngles = styled.img`
+display: none;
+@media (min-width: ${props => props.theme.breakpoints.tablet}) {
+  display: inline-block;
+  width: 24%;
+  height: 300px;
+  margin: 0 26px 0 0;
+}
+`
+
+const GownsPage = () => (
+  <Layout>
+    <NavBar />
+    <LogoContainer>
+      <Logo notag />
+    </LogoContainer>
+    <NavMenuDesktop />
+    <BodyContainer>
+    <SearchMenuContainer>
+      <TitleContainer>
+        <LeftArrowContainer>
+          <Icons name={["fas", "arrow-left"]} size="1x" />
+        </LeftArrowContainer>
+        <CollectionTitle>{collectionTitle[0]}</CollectionTitle>
+        <RightArrowContainer>
+          <Icons name={["fas", "arrow-right"]} size="1x" />
+        </RightArrowContainer>
+      </TitleContainer>
+      <DesktopCollectionTitleContainer>
+        <CollectionTitle>Fall 2020</CollectionTitle>
+      </DesktopCollectionTitleContainer>
+    </SearchMenuContainer>
+    <ImageAndDetailsContainer>
+    <GownImage src={cardData[0].image} />
+    <DesktopFlexColumn>
+    <Name>{cardData[0].mobileLabel}</Name>
+    <Brief>Stuff</Brief>
+    <DetailsFeatures>Details and Features</DetailsFeatures>
+    <Description>Description with a whole bunch of stuff describing the gown</Description>
+    <DesktopButtonContainer>
+      <PinkButton
+        label="See More From This Collection"
+        width={`${"250px"}`}
+        fontsize={`${"12px"}`}
+        height={`${"14px"}`}
+        weight={`${"800"}`}
+        padding={`${"5px 0"}`}
+        shadow={`${"0px 4px 4px rgba(0, 0, 0, 0.25)"}`}
+        color={`${"#000000"}`}
+        desktopfontsize={`${"14px"}`}
+        desktopheight={`${"16px"}`}
+      />
+    </DesktopButtonContainer>
+    </DesktopFlexColumn>
+    <DetailsAndFeaturesContainer>
+      <Icons name={["far", "plus-square"]} />
+      <Details>Details and Features</Details>
+    </DetailsAndFeaturesContainer>
+    </ImageAndDetailsContainer>
+    <MobileButtonContainer>
+      <PinkButton
+        label="See More From This Collection"
+        width={`${"250px"}`}
+        fontsize={`${"12px"}`}
+        height={`${"14px"}`}
+        weight={`${"800"}`}
+        padding={`${"5px 0"}`}
+        shadow={`${"0px 4px 4px rgba(0, 0, 0, 0.25)"}`}
+        color={`${"#000000"}`}
+        desktopfontsize={`${"14px"}`}
+        desktopheight={`${"16px"}`}
+      />
+    </MobileButtonContainer>
+    <DifferentAnglesContainer>
+    <DifferentAngles src={cardData[1].image} />
+    <DifferentAngles src={cardData[2].image} />
+    <DifferentAngles src={cardData[3].image} />
```

Comment:
The dimensions on these images make them look a little bit skewed.

Reply:
done